### DISCUSSION TITLE
Please help me! My cluster has been deployed, but venus-sealer has not yet assigned tasks.

### DISCUSSION THREAD

**Discussion by Sela-Hao** [filecoin-project/venus#4479] [Q&A]
Excuse me, the cluster has been deployed, but venus-sealer has not assigned tasks. What should I do?

**Comment by Joss-Hua** [comment]
It's probably a configuration problem. You can try, on the *venus-auth*, use the command of `./Venus messenger wallet list` to check whether the cluster configuration information is wrong.

**Comment by simlecode** [comment]
Please show the detailed log

**Comment by diwufeiwen** [comment]
> Excuse me, the cluster has been deployed, but venus-sealer has not assigned tasks. What should I do?

Please execute the command： ./venus-sealer sectors pledge，   Every time it is executed, it will start to make a sector.

[root@200-17 thelitao]# ./venus-sealer sectors pledge
Created CC sector:  2
[root@200-17 thelitao]# ./venus-sealer sectors list
ID  State     OnChain  Active  Expiration  Deals  
1   WaitSeed  NO       NO      n/a         CC     
2   Packing   NO       NO      n/a         CC  

In addition, please use the latest venus-sealer v1.2.0

**Reply by Joss-Hua** [reply]
✋   [venus-sealer v1.2.0 ]( https://github.com/filecoin-project/venus-sealer/releases/tag/v1.2.0)

**Comment by diwufeiwen** [comment]
> Excuse me, the cluster has been deployed, but venus-sealer has not assigned tasks. What should I do?

venus 0.9.8 fix beacon randomness (#4475), doc:https://venus.filecoin.io/guide/How-To-Deploy-MingPool.html#install-venus-sealer. If you encounter any problems with building a cluster according to the doc, you can raise them here, and we will reply in time
